janus_server/src/bin/janus_cli.rs

```diff
@@ -0,0 +1,307 @@
+use anyhow::{Context, Result};
```

Comment:
I suggest using [`color-eyre`](https://docs.rs/color-eyre/latest/color_eyre/) so we get super pretty errors printed to stderr.

Reply:
Unfortunately, it looks like `color-eyre` requires use of its `color_eyre::Result` type to get the nicely-colored output. This is incompatible with `janus_main` right now since `janus_main` requires use of an `anyhow::Result`. It wouldn't be too hard of a refactor, but I've already decided to leave a larger `janus_main` refactoring for later -- I'll let this get picked up in #315.